### ISSUE TITLE
3.2.1 - Previous Customization not working after updating using Sage/Platform/Mobile/GroupedList

### ISSUE THREAD

**Issue by rjledger** [Saleslogix/argos-saleslogix#331]
We have a custom all our customers use where we change Activity:
     list.js
     MyList.js
to incorporate GroupedList doing a GroupBySections rather than a "normal list":
        // CUSTOM - Forces grouping - 20141111 - rjl
        //inherited by myList.js as well!
        getGroupBySections: function() {
            var groupBySections = [
              {
                id: 'section_StartDate',
                description: 'Start Date',
                section: new DateTimeSection({ groupByProperty: 'StartDate', sortDirection: 'desc' })
              }
            ];
            return groupBySections;
        }
        //END of CUSTOM

and it's no longer working. Could not see anything that changed that could have broken it

-- to add:
Basically if I change to using 'Sage/Platform/Mobile/GroupedList' Firebug throws up a ton of console errors about:
   Get Format.js
   Get Template.js
   Get Action.js
   Get Validator.js
   Get Application.js
   Get Home.js
   Get LeftDrawer.js
   Get List.js
   Get Environment.js
All of a sudden the "path(s)" are messed up!  
.. all things "good" look like:
       http://"my site"/informoble32/whatever...
.. all things "bad" look like:
       http://"my site"/Mobile/whatever.

All of this worked w/3.2.0

**Comment by jbest84** [comment]
Are you using the develop branch or version 3.2.1? Because the difference from 3.2 to 3.2.1 (mentioned in your title) are very minor: https://github.com/Saleslogix/argos-sdk/compare/v3.2...jbest84:v3.2.1

**Comment by jbest84** [comment]
The develop branch has some changes that are required to make in your development.html file (production requires no change): https://github.com/Saleslogix/argos-saleslogix/wiki/3.3-Changes

**Comment by rjledger** [comment]
A - not using Dev branch - using version 3.2.1
B - This is failing after being deployed by the AA so am not using the dev html.. using .aspx

**Comment by jbest84** [comment]
I just tried this on 3.2.1 and the latest develop. I cannot reproduce the errors you are seeing.

**Comment by rjledger** [comment]
OK.. I just hit my install w/a BIG hammer (reboot(s), etc and it finally cleared up and is working.
